.github/workflows/ci.yml

```diff
@@ -1606,9 +1606,13 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: "Install pyenv"
-        uses: "gabrielfalcao/pyenv-action@v18"
-        with:
-          default: 3.9.7
```

Comment:
At the time of this writing, [`gabrielfalcao/pyenv-action@v18`](https://github.com/gabrielfalcao/pyenv-action/) hasn't been updated in about 10 months.